models/repo.go

```diff
@@ -656,8 +658,12 @@ func GetMirror(repoId int64) (*Mirror, error) {
 }
 
 func updateMirror(e Engine, m *Mirror) error {
-	_, err := e.Id(m.ID).Update(m)
```

Comment:
to be honest, these are my first steps with Go and XORM. For some reason this update() ignores the new column added in the Mirror model (enable_prune) for this reason I included the extra update instruction.

I could suggest a review for a more avances Go programmer .